src/components/Form/FormFormats.js

```diff
@@ -1119,46 +1195,114 @@ export default {
+                $description: String, $instructor:ID, $isConfirmed:Boolean, $maxCapacity: Int, $totalTuition: Decimal,
+                $title: String!) {
+            createCourse(
+                  id: $id,
+                  title: $title,
+                  description:$description,
+                  courseType: CLASS,
+                  academicLevel: $academicLevel,
+                  instructor: $instructor,
+                  startDate: $startDate,
+                  endDate: $endDate,
+                  room: "Stanford Room",
+                  maxCapacity: $maxCapacity,
+                  courseCategory: $courseCategory,
+                  totalTuition: $totalTuition,
+                  isConfirmed: $isConfirmed,
+                  availabilities: $availabilities
+              ) {
+                  created
+                  course {
+                      academicLevelPretty
+                      id
+                      title
+                      description
+                      courseType
+                      academicLevel
+                      startDate
+                      endDate
+                      instructor {
+                          user {
+                              firstName
+                              lastName
+                          }
+                      }
+                      }
+                  }
+              }
             `;
 
-            const { courseInfo, tuition } = formData;
+            const { courseDescription, dayAndTime, tuition  } = formData;
+            const availabilities = (() => {
+                const availabilityList = [
+                    {
+                        "dayOfWeek": dayAndTime.weekday1,
+                        "startTime": dayAndTime.startTime1.format("HH:mm"),
+                        "endTime":  dayAndTime.endTime1.format("HH:mm"),
+                    }
+                ]
+                if (dayAndTime.weekday2 && dayAndTime.startTime2 && dayAndTime.EndTime2){
+                    availabilities.push(
+                        {
+                            "dayOfWeek": dayAndTime.weekday2,
+                            "startTime": dayAndTime.startTime2 ? dayAndTime.startTime2.format("HH:mm") : null,
+                            "endTime": dayAndTime.endTime2 ? dayAndTime.endTime2.format("HH:mm") : null,
+                        }
+                    );
+                }
+                if (dayAndTime.weekday3 && dayAndTime.startTime3 && dayAndTime.EndTime3) {
+                    availabilities.push(
+                        {
+                            "dayOfWeek": dayAndTime.weekday3,
+                            "startTime": dayAndTime.startTime3 ? dayAndTime.startTime3.format("HH:mm"): null,
+                            "endTime": dayAndTime.endTime3 ? dayAndTime.endTime3.format("HH:mm"): null
+                        }
+                    );
+                }
```

Comment:
We should try to convert this into a more adaptable method and not just hard-coded to 3 availabilities maximum, probably by using an array or something similar.

Reply:
good catch but, i'm not spending time on doing it for this PR. pushing this off to #670